packages/devtools_app/lib/src/service/isolate_manager.dart

```diff
@@ -20,70 +19,70 @@ import 'vm_service_wrapper.dart';
-  VmServiceWrapper _service;
+  VmServiceWrapper? _service;
 
-  final StreamController<IsolateRef> _isolateCreatedController =
-      StreamController<IsolateRef>.broadcast();
-  final StreamController<IsolateRef> _isolateExitedController =
-      StreamController<IsolateRef>.broadcast();
+  final StreamController<IsolateRef?> _isolateCreatedController =
+      StreamController<IsolateRef?>.broadcast();
+  final StreamController<IsolateRef?> _isolateExitedController =
+      StreamController<IsolateRef?>.broadcast();
 
-  ValueListenable<IsolateRef> get selectedIsolate => _selectedIsolate;
-  final _selectedIsolate = ValueNotifier<IsolateRef>(null);
+  ValueListenable<IsolateRef?> get selectedIsolate => _selectedIsolate;
+  final _selectedIsolate = ValueNotifier<IsolateRef?>(null);
 
   int _lastIsolateIndex = 0;
-  final Map<String, int> _isolateIndexMap = {};
+  final Map<String?, int> _isolateIndexMap = {};
 
-  ValueListenable<List<IsolateRef>> get isolates => _isolates;
+  ValueListenable<List<IsolateRef?>> get isolates => _isolates;
   final _isolates = ListValueNotifier(const <IsolateRef>[]);
 
-  Stream<IsolateRef> get onIsolateCreated => _isolateCreatedController.stream;
+  Stream<IsolateRef?> get onIsolateCreated => _isolateCreatedController.stream;
 
-  Stream<IsolateRef> get onIsolateExited => _isolateExitedController.stream;
+  Stream<IsolateRef?> get onIsolateExited => _isolateExitedController.stream;
 
-  ValueListenable<IsolateRef> get mainIsolate => _mainIsolate;
-  final _mainIsolate = ValueNotifier<IsolateRef>(null);
+  ValueListenable<IsolateRef?> get mainIsolate => _mainIsolate;
+  final _mainIsolate = ValueNotifier<IsolateRef?>(null);
 
-  final _isolateRunnableCompleters = <String, Completer<void>>{};
+  final _isolateRunnableCompleters = <String?, Completer<void>>{};
 
   Future<void> init(List<IsolateRef> isolates) async {
     // Re-initialize isolates when VM developer mode is enabled/disabled to
     // display/hide system isolates.
     addAutoDisposeListener(preferences.vmDeveloperModeEnabled, () async {
       final vmDeveloperModeEnabled = preferences.vmDeveloperModeEnabled.value;
-      final vm = await serviceManager.service.getVM();
+      final vm = await serviceManager.service!.getVM();
       final isolates = [
-        ...vm.isolates,
-        if (vmDeveloperModeEnabled) ...vm.systemIsolates,
+        ...vm.isolates!,
+        if (vmDeveloperModeEnabled) ...vm.systemIsolates!,
       ];
-      if (selectedIsolate.value.isSystemIsolate && !vmDeveloperModeEnabled) {
+      if (selectedIsolate.value!.isSystemIsolate! && !vmDeveloperModeEnabled) {
         selectIsolate(_isolates.value.first);
       }
       await _initIsolates(isolates);
     });
     await _initIsolates(isolates);
   }
 
-  IsolateState get mainIsolateDebuggerState {
-    return _isolateStates[_mainIsolate.value];
+  IsolateState? get mainIsolateDebuggerState {
+    return _isolateStates[_mainIsolate.value!];
```

Comment:
return _mainIsolate.value != null ? _isolateStates[_mainIsolate.value!] : null

same with isolateDebuggerState and selectedIsolateState below

Reply:
done